pkg/apmintegration/file.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ava-labs/apm/types"
+	"github.com/ava-labs/avalanche-cli/pkg/application"
+	"gopkg.in/yaml.v3"
+)
+
+func GetRepos(app *application.Avalanche) ([]string, error) {
+	repositoryDir := filepath.Join(app.ApmDir, "repositories")
+	orgs, err := os.ReadDir(repositoryDir)
+	if err != nil {
+		return []string{}, err
+	}
+
+	output := []string{}
+
+	for _, org := range orgs {
+		repoDir := filepath.Join(repositoryDir, org.Name())
+		repos, err := os.ReadDir(repoDir)
+		if err != nil {
+			return []string{}, err
+		}
+		for _, repo := range repos {
+			output = append(output, org.Name()+"/"+repo.Name())
+		}
+	}
+
+	return output, nil
+}
+
+func GetSubnets(app *application.Avalanche, repoAlias string) ([]string, error) {
+	subnetDir := filepath.Join(app.ApmDir, "repositories", repoAlias, "subnets")
+	subnets, err := os.ReadDir(subnetDir)
+	if err != nil {
+		return []string{}, err
+	}
+	subnetOptions := make([]string, len(subnets))
```

Comment:
why is this called subnetOptions?

Reply:
Because we're letting the user choose which subnet they want